cluster_provisioning/ansible/playbooks/install-couchbase-server-package.yml

```diff
@@ -0,0 +1,46 @@
+---
+- hosts: cb_pool
+  remote_user: root
+  any_errors_fatal: true
+  serial: 1
+  vars:
+    couchbase_server_primary_node: "{{ groups['cb_pool'][0] }}"
+  connection: local
+  tasks:
+    - shell: echo {{ couchbase_server_primary_node }} > ../../temp_couchbase_server_primary_node
+
+- hosts: cb_pool
+  remote_user: root
+  any_errors_fatal: true
+  sudo: true
+  vars:
+    couchbase_server_package_base_url:
+    couchbase_server_package_name:
+    couchbase_server_package_url: "{{ couchbase_server_package_base_url }}/{{ couchbase_server_package_name }}"
+
+    couchbase_server_admin: Administrator
+    couchbase_server_password: password
+    couchbase_server_ram: "{{ ansible_memtotal_mb|int }}"
+    couchbase_server_admin_port: 8091
+    couchbase_server_home_path: /opt/couchbase
+
+    couchbase_server_node: "{{ couchbase_server_primary_node }}"
+    couchbase_server_bucket_name: default
+    couchbase_server_bucket_name2: bucket-1
```

Comment:
My preference would an array of bucket names. In future we might need to create more buckets.